scripts/package.json

```
@@ -19,6 +19,7 @@
     "release:is-pr-frozen": "ts-node --swc ./release/is-pr-frozen.ts",
     "release:is-prerelease": "ts-node --swc ./release/is-prerelease.ts",
     "release:is-version-published": "ts-node --swc ./release/is-version-published.ts",
+    "release:pick-patches": "node --require esbuild-register ./release/pick-patches.ts",
```

Can we do this like the rest of the scripts, or is that not possible/bad for some reason?

```suggestion
    "release:pick-patches": "ts-node --swc ./release/pick-patches.ts",
```

---

I just had some problems getting the debugger attached with ts-node. But let's be consistent indeed.